src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -34,6 +37,33 @@ namespace configuration
         /// seconds.
         std::time_t ParseTimeUnit(const std::string& option) const;
 
+        /// @brief The groups information
```

Comment:
Brief should possible be something more like "function to get the groups information" and whatever other details you see necessary.

Reply:
Done!